src/ergw_aaa_gx.erl

```diff
@@ -277,23 +279,23 @@ handle_cca({error, no_connection}, Session, Events,
     Avps = maps:get(Answer, Answers, #{'Result-Code' =>
 					   [?'DIAMETER_BASE_RESULT-CODE_AUTHORIZATION_REJECTED']}),
     handle_cca(['CCA' | Avps], Session, Events, Opts, State#state{state = peer_down});
-handle_cca({error, no_connection}, Session, Events,
+handle_cca({error, timeout}, Session, Events,
```

Comment:
Is that change correct?

Reply:
Yes, it is a fixing a wrong function clause match, however certainly has nothing to do with this feature. Shall I "unfix" it and make a new PR for clarity?